src/base/static/client/place-tags.js

```diff
@@ -0,0 +1,70 @@
+const createPlaceTag = async (url, tagData) => {
+  try {
+    const response = await fetch(`${url}/tags`, {
+      headers: {
+        "Content-Type": "application/json",
+      },
+      credentials: "include",
+      method: "POST",
+      body: JSON.stringify(tagData),
+    });
+
+    if (response.status < 200 || response.status >= 300) {
+      throw new Error(response.statusText);
+    }
+
+    return await response.json();
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.error("Error: Failed to create Place tag.", err);
+  }
+};
+
+const updatePlaceTag = async (url, newData) => {
+  try {
+    const response = await fetch(url, {
+      headers: {
+        "Content-Type": "application/json",
+      },
+      credentials: "include",
+      method: "PATCH",
+      body: JSON.stringify(newData),
+    });
+
+    if (response.status < 200 || response.status >= 300) {
+      throw new Error(response.statusText);
```

Comment:
There might be a bug here where this code is being called, because we are using `if (response)` to check whether the request was successful. Instead, I think we should `console.error(response.statusText); return null` here, instead of throwing an error. 

Also, I think the `if (response)` is a better control flow, because it's technically not an error, and throwing errors is generally not a good practice for control flow.